docs/content/parent-child-1-ui-init.wsd

```diff
@@ -0,0 +1,24 @@
+@startuml "parent-child-1-ui-init"
+' © Sebastian Porto 2016, 
+' licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License.
+autonumber "<b>(#)"
+participant User #white
```

Comment:
If you replace `participant` with `actor` it will show up as a stickman.

Reply:
Done